UnityProject/Assets/Prefabs/Items/Clothing/Headwear/Hardhats/HardHat.prefab

```diff
@@ -18,7 +18,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 0
+  m_IsActive: 1
```

Comment:
what did you activate here?

Reply:
No clue, unity might have detected an outdated value and decided to update it on the file itself.